fast64_internal/oot/collision/exporter/to_c/collision.py

```diff
@@ -309,7 +321,6 @@ def exportCollisionToC(
 
             # write file
             path = ootGetPath(exportPath, isCustomExport, "assets/objects/", folderName, False, True)
-            filename = exportSettings.filename if exportSettings.isCustomFilename else f"{name}_collision"
             writeCData(colData, os.path.join(path, f"{filename}.h"), os.path.join(path, f"{filename}.c"))
             if not isCustomExport:
                 addIncludeFiles(folderName, path, name)
```

Comment:
Game wouldn´t compile without this, I assume this is a bug?
```suggestion
                addIncludeFiles(folderName, path, filename)
```

Reply:
should work properly now (with latest decomp)